src/pages/Creatures.tsx

```diff
@@ -248,17 +202,7 @@ export default function Creatures() {
             ))}
           </TableBody>
           <TableFooter>
-            <TableRow>
-              <TablePagination
-                style={{ borderTop: "1px solid rgba(224, 224, 224, 1)" }}
-                count={count}
-                page={query.page}
-                labelRowsPerPage="Num: "
-                onPageChange={pageChange}
-                rowsPerPage={query.size}
-                onRowsPerPageChange={sizeChange}
-              />
-            </TableRow>
+            <TableRow>{InstanceTablePagination}</TableRow>
```

Comment:
Lets still pass the specific props here so we don't couple the hook to the component too much.

You can make code so DRY it becomes brittle.